doc/README.cuda_support.md

```diff
@@ -105,7 +120,8 @@ must match the version of Open-CE.
 
 | Open-CE version | TensorRT version | file type |
 |-----------------|------------------|-----------|
-| 1.0 (marmotini) | 7.0.0.1          |  tar.gz   |
+| 1.0 (marmotini) | 7.0.0.11         |  tar.gz   |
+| 1.1 (unnamed)   | 7.2.*            |  tar.gz   |
```

Comment:
Maybe we need to rework this table. Because if you build 1.1 against cuda 10.2, you'll still need TRT 7.0.

Reply:
Done.